modules/transport/index.js

```diff
@@ -0,0 +1,59 @@
+// FIXME: change to '../API' when we update to webpack2. If we do this now all
+// files from API modules will be included in external_api.js.
+import { API_ID } from '../API/constants';
+import { getJitsiMeetGlobalNS } from '../util/helpers';
+
+import Transport from './Transport';
+import PostMessageTransportBackend from './PostMessageTransportBackend';
+
+export {
+    Transport,
+    PostMessageTransportBackend
+};
+
+/**
+ * Option for the default low level transport.
+ *
+ * @type {Object}
+ */
+const postisOptions = {};
+
+if (typeof API_ID === 'number') {
+    postisOptions.scope = `jitsi_meet_external_api_${API_ID}`;
```

Comment:
I had some thoughts when reading this.  How about we change it for the new API? That way we know that we don't get old API methods / events in the new scope.